tests/test_db_service.py

```diff
@@ -38,7 +39,7 @@ def test_db_services_share_mongodb_client(self):
         """Test client is shared across different db_service_objects."""
         foo = DBService("foo", self.client)
         bar = DBService("bar", self.client)
-        assert foo.db_client is bar.db_client
+        self.assertEqual(foo.db_client, bar.db_client)
```

Comment:
```suggestion
        self.assertIs(foo.db_client, bar.db_client)
```
moving `is` into `equal` does not seem like the right way here